addons/account/models/account_bank_statement_line.py

```diff
@@ -743,8 +743,12 @@ def _synchronize_from_moves(self, changed_fields):
                         'amount': liquidity_lines.balance,
                     })
 
-                if len(suspense_lines) == 1:
-
+                if len(suspense_lines) > 1:
+                    raise UserError(_(
+                        "The %s reached an invalid state regarding its related statement line.\n"
```

Comment:
The message doesn't seem to be in correct English 
```suggestion
                        "%s reached an invalid state regarding its related statement line.\n"
```